app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminNewsletterCreateNewTemplateActionGroup.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminNewsletterCreateNewTemplateActionGroup">
+        <annotations>
+            <description>Create New Template</description>
+        </annotations>
+        <arguments>
+            <argument name="template" defaultValue="_defaultNewsletter" type="entity"/>
+        </arguments>
+        <amOnPage url="{{NewsletterTemplateForm.url}}" stepKey="amOnNewsletterTemplatePage"/>
+        <waitForPageLoad stepKey="waitForPageLoad1"/>
```

Comment:
I personally prefer split that into more atomic parts like:
`AdminNewsletterFromOpenActionGroup`
That is because you may want to open EDIT (not new) Page, then you use different FromOpen Action Group, the rest ist just the same. We improve reusability this way.